test/testcafe/spec/IdentifyWithRememberUsername_spec.js

```diff
@@ -0,0 +1,239 @@
+import xhrSuccessWithInteractionCode from '../../../playground/mocks/data/idp/idx/success-with-interaction-code.json';
+import xhrIdentifyWithPassword from '../../../playground/mocks/data/idp/idx/identify-with-password';
+import emailVerification from '../../../playground/mocks/data/idp/idx/authenticator-verification-email';
+import xhrSuccessTokens from '../../../playground/mocks/data/oauth2/success-tokens';
+
+const identifyWithError = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(xhrIdentify)
+  .onRequestTo('http://localhost:3000/idp/idx/identify')
+  .respond(xhrPassword)
+  .onRequestTo('http://localhost:3000/idp/idx/challenge/answer')
+  .respond(xhrErrorIdentify, 403);  
+
+const identifyWithPasswordError = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(xhrIdentifyWithPassword)
+  .onRequestTo('http://localhost:3000/idp/idx/identify')
+  .respond(xhrErrorIdentify, 403);  
+
+const identifyMock = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(xhrIdentify)
+  .onRequestTo('http://localhost:3000/idp/idx/identify')
+  .respond(xhrPassword)
+  .onRequestTo('http://localhost:3000/idp/idx/challenge/answer')
+  .respond(xhrSuccess);
+
+const identifyWithPasswordMock = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(xhrIdentifyWithPassword)
+  .onRequestTo('http://localhost:3000/idp/idx/identify')
+  .respond(xhrSuccess);
+
+const identifyWithEmailAuthenticator = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(xhrIdentify)
+  .onRequestTo('http://localhost:3000/idp/idx/identify')
+  .respond(emailVerification)
+  .onRequestTo('http://localhost:3000/idp/idx/challenge/answer')
+  .respond(xhrSuccessWithInteractionCode)
+  .onRequestTo('http://localhost:3000/oauth2/default/v1/token')
+  .respond(xhrSuccessTokens);
+
+const identifyWithEmailAuthenticatorError = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(xhrIdentify)
+  .onRequestTo('http://localhost:3000/idp/idx/identify')
+  .respond(emailVerification)
+  .onRequestTo('http://localhost:3000/idp/idx/challenge/answer')
+  .respond(xhrErrorIdentify, 403);
+
+const identifyRequestLogger = RequestLogger(
+  /idx\/identify|\/challenge/,
+  {
+    logRequestBody: true,
+    stringifyRequestBody: true,
+    logRequestHeaders: true,
+  }
+);
+
+fixture('Identify With Remember Username');
+
+async function setup(t, options) {
+  const identityPage = new IdentityPageObject(t);
+  await identityPage.navigateToPage(options || {});
+  return identityPage;
+}
+
+test.requestHooks(identifyRequestLogger, identifyWithError)('identifer first flow - should NOT remember username after failed authentication', async t => {
+  const identityPage = await setup(t);
+  await rerenderWidget({
+    features: {
+      rememberMe: true,
+    }
```

Comment:
a test for `'features.rememberMe': true`, if this is supposed to support this configuration key

Reply:
I don't follow here, we're essentially turning the feature on here already. The SIW has code to flatten the object settings structure so that will ultimately become "features.rememberMe", but it's not the intention of this spec to test that (not sure if that's what you're asking)